Makefile

```diff
@@ -19,8 +19,13 @@ else
 	BUILD_ARG=--build-arg=SANDBOX_IMAGE_TAG=$(TAG)
 endif
 
-.PHONY: all
-all: docker_build_all
+
+#
+# This recipe builds and pushes images for production. Note: RELEASE_TAG must be set
+#
+.PHONY: cloudbuild
+cloudbuild: docker_push_all
```

Comment:
It might be worth having another target that ensures `RELEASE_TAG` is set that is used to avoid errors?

e.g:
```
ensure_release_tag_exists:
ifndef RELEASE_TAG
	$(error RELEASE_TAG must be set)
endif
```

Reply:
Ah I was wondering how to do this! Thanks, I'll add it in